odl/trafos/backends/pywt_bindings.py

```diff
@@ -275,6 +338,25 @@ def pywt_coeffs_from_flat_array(arr, shapes):
         ``Di`` the tuple of i-th level detail coefficient arrays. Each of
         the ``Di`` tuples has length ``2 ** ndim - 1``, where ``ndim`` is
         the number of dimensions of ``arr``.
+
+    Examples
+    --------
+    Turn the flat array from the example in `pywt_flat_array_from_coeffs`
+    back into a coefficient list:
+
+    >>> arr = [1, 1, 2, 2, 3, 3, 4, 4]
+    >>> # approximation and detail coefficient shapes
+    >>> shapes = [(1, 2), (1, 2)]
+    >>>
+    >>> coeffs = pywt_coeffs_from_flat_array(arr, shapes)
+    >>> approx, details = coeffs
+    >>> print(approx)
+    [[1 1]]
+    >>> for detail in details:
```

Comment:
What about [this place](https://github.com/odlgroup/odl/pull/547/files/66fb05044a8a6dccc70f584baed6e0abdaecc39e#diff-a57dd4d9e768f69a8e6f15d547ed456cR438)? The output of the direct print will look like this:
```
(array([[ 0.5,  0.5],
       [-0.5,  0.5]]), array([[ 0.5,  0.5],
       [ 0.5,  0.5]]), array([[-0.5,  0.5],
       [-0.5,  0.5]]))
```
Very ugly IMO.